stl/inc/atomic

```diff
@@ -427,6 +457,45 @@ struct _Atomic_storage {
         return _Result;
     }
 
+#if _HAS_CXX20
+    void wait(_Ty _Expected, const memory_order _Order = memory_order_seq_cst) const noexcept {
+        const auto _Storage_ptr = _STD addressof(_Storage);
+        auto _Expected_ptr      = _STD addressof(_Expected);

```

Comment:
While you use `memcpy` to write through `_Expected_ptr`, it appears that it's never reassigned, so it can be top-level `const`:
```suggestion
        const auto _Storage_ptr  = _STD addressof(_Storage);
        const auto _Expected_ptr = _STD addressof(_Expected);
```